v21.1/import-performance-best-practices.md

```diff
@@ -123,6 +123,18 @@ CSV DATA (
 
 This method has the added benefit of alerting on potential issues with the import sooner; that is, you will not have to wait for the file to load both the schema and data just to find an error in the schema.
 
+### Import into a schema with secondary indexes
+
+Importing data into a schema that has [secondary indexes](schema-design-indexes.html) will increase the import's total time to completion. The import job will perform the import and populate the indexes concurrently, which causes the longer import time.
```

Comment:
I might flip these around: When importing data into a table with secondary indexes, required secondary index data is derived and ingested concurrently with the ingestion of the table data. This *may* result in an import that takes longer than if the table did not have secondary indexes, but typically adds less time to the initial import than following it will a separate index addition pass. Thus, importing tables with their secondary indexes is the default workflow, suitable for more import jobs.

However, in **large** import jobs, just importing table data, then adding indexes, may be preferable for the increased visibility into its progress and ability to retry each step independently.